schema/codelists/partyRole.csv

```diff
@@ -1,21 +1,21 @@
-contractImplementationManager,Contract implementation manager,The organization responsible for managing the implementation of the contract on behalf of the buyer. (This may be different from the procuring entity that manages the contracting process.),,
+contractImplementationManager,Contract implementation manager,The organization responsible for managing the implementation of the contract on behalf of the buyer. (This may be different from the procuring entity that manages the contracting (or planning) process.),,
 mediationBody,Mediation body,The organization responsible for mediation procedures.,,
-processContactPoint,Process contact point,A contact point dedicated to this contracting process.,,
+processContactPoint,Process contact point,A contact point dedicated to this contracting (or planning) process.,,
 reviewContactPoint,Review contact point,The service from which information about the review procedure can be obtained.,,
-informationService,Information service,An organization that can provide information regarding specific aspects of the contracting process.,,
+informationService,Information service,An organization that can provide information regarding specific aspects of the contracting (or planning) process.,,
 privateParty,Private party,"The counter party of the procuring authority in the public-private partnership (PPP) contract. A private organization which has been granted the contract to construct and operate a government asset, and which is usually created under the form of a special purpose vehicle (SPV).",https://ppp-certification.com/ppp-certification-guide/glossary,
 leadBank,Lead bank,A lead bank is a bank that oversees the arrangement of a loan syndication. This role typically relates to financing a public-private partnership project.,https://www.investopedia.com/terms/l/lead-bank.asp,
 lender,Lender,"An institution that finances the contracting process through debt financing, like traditional debt or project bonds. This role typically relates to financing a public-private partnership project.",https://ppp-certification.com/ppp-certification-guide/glossary,
 equityInvestor,Equity investor,"An investor that finances the contracting process through equity financing: that is, capital in exchange for a share of ownership. This role typically relates to financing a public-private partnership project.",https://ppp-certification.com/ppp-certification-guide/glossary,
-notifiedPotentialSupplier,Notified potential supplier,"A potential supplier that has been notified about the contracting process: for example, through a non-exclusive invitation to bid.",,
+notifiedPotentialSupplier,Notified potential supplier,"A potential supplier that has been notified about the contracting (or planning) process: for example, through a non-exclusive invitation to bid.",,
```

Comment:
Are potential suppliers notified about planning processes?

Reply:
Sure? "Dear company that participated in previous computer procedure / company that responsed to the prior market consultation, here is a link to our PIN about the procedures we are planning to run next year. Get ready & happy for your feedback."

Reply:
Sounds good. I guess they are "potential" in terms of the eventual contracting process.